packages/firestore/src/local/index_manager.ts

```diff
@@ -50,4 +53,86 @@ export interface IndexManager {
     transaction: PersistenceTransaction,
     collectionId: string
   ): PersistencePromise<ResourcePath[]>;
+
+  /**
+   * Adds a field path index.
+   *
+   * Values for this index are persisted asynchronously. The index will only be
```

Comment:
This is referring to backfill right? If so, we should probably give a high level explanation of backfilling, instead of saying `persisted async`.

Reply:
Improved comment.